mobile/src/components/common/input-secure/input-secure.tsx

```diff
@@ -46,7 +46,7 @@ const InputSecure = <T extends FormControlValues>({
       />
       <Pressable style={styles.button} onPress={handleContentVisabilityToggle}>
         <Icon
-          name={isContentVisible ? 'visibilityOff' : 'visibility'}
+          name={isContentVisible ? 'visibility' : 'visibilityOff'}
```

Comment:
This change should be not there, we should use visibilityOff when isContentVisible, you should change it in isSecure prop

Reply:
I don't understand why not here. QA wants the eye icon to be reversed

Reply:
Yes it means when isContentVisible we should see visibilityOff, the proplem is that isContentVisible has reverted value to actual state, it's because there is `isSecure={isContentVisible}` but should be `isSecure={!isContentVisible}`

Reply:
if i set isSecure={!isContentVisible} i need to change useState from useState(true) to useState(false) bcos we need to pass into Input component always true to set secureTextEntry="true"

Reply:
It is necessary to pass true to Input when initializing InputSecure to set secureTextEntry="true" otherwise text is visible while you type